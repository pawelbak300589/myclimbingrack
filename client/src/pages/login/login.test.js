import React from 'react';
import { shallow } from "enzyme";

import LoginPage from "./login.component";

it('should render LoginPage component', () => {
    expect(shallow(<LoginPage />)).toMatchSnapshot();
});