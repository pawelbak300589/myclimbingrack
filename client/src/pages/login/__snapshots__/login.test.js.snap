// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginPage component 1`] = `ShallowWrapper {}`;
